lua/starfall/libs_sh/holograms.lua

```diff
@@ -332,45 +356,68 @@ else
 		end
 	end
 
+	local function clearParentFix(ent)
+		if ent.sf_parent then
+			ent.sf_parent.sf_children[ent] = nil
+			ent.sf_parent = nil
+		end
+	end
+	
+	local function setParentFix(ent, parent, bone, func)
+		if not parent.sf_children then
+			parent.sf_children = {}
```

Comment:
Should this use `SF.EntityTable`? It will only ever contain hologram entities.

Reply:
Yeah I think that would be best and it should work fine in this case.

Reply:
Or you can call clearParentFix in the removeHolo functions

Reply:
These can be moved above the `return function(instance)` as well

Reply:
`hologramOnDestroy` I mean is where the clearParentFix should go